pkg/config/config.go

```diff
@@ -588,6 +588,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("kubernetes_node_labels_as_tags", map[string]string{})
 	config.BindEnvAndSetDefault("kubernetes_node_annotations_as_tags", map[string]string{"cluster.k8s.io/machine": "kube_machine"})
 	config.BindEnvAndSetDefault("kubernetes_node_annotations_as_host_aliases", []string{"cluster.k8s.io/machine"})
+	config.BindEnvAndSetDefault("kubernetes_node_labels_as_cluster_name", map[string]string{})
```

Comment:
Should we allow multiple labels? In the end, we only have 1 cluster name?

Reply:
it was to ease possible migration from a label to another. but it is not mandatory. I can change to only one label key.